src/st/wrp/data.c

```diff
@@ -0,0 +1,24 @@
+#include "types.h"
+
+/*
+ * n.b.! this file is used for uninitialized data (.bss)
+ *       referenced throughout the overlay.
+ */
+
+// the room the player will be warping to
+/* 0x80193AA0 */ s32 DestinationWarpRoom;
```

Comment:
Your variables need g_ according to the naming convention https://github.com/Xeeynamo/sotn-decomp/blob/master/docs/STYLE.md#naming

Reply:
Moved them into `warp.c` and made them static.